Tools/setup.csh

```diff
@@ -103,8 +111,18 @@ if ( -f $LEGTOPTAGGERFILE && -l $LEGTOPTAGGERFILE ) then
     rm $LEGTOPTAGGERFILE
 endif
 
+# -p give no error if directory exists
+mkdir -p $STOPCFGDIR
+# -o for overwrite softlinks if they exist
+${SRC}/SusyAnaTools/Tools/scripts/getStopCfg.sh -o -t supp_cfg_tag -d $STOPCFGDIR
+
+# -p give no error if directory exists
 mkdir -p $TAGGERCFGDIR
-${SRC}/TopTagger/Tools/getTaggerCfg.sh -t MVAAK8_Tight_v1.2.1 -d $TAGGERCFGDIR
-${SRC}/TopTagger/Tools/getTaggerCfg.sh -t Legacy_AK4Only_v0.1.1 -f $LEGTOPTAGGERFILE -d $TAGGERCFGDIR
-${SRC}/TopTagger/Tools/getTaggerCfg.sh -t MVAAK8_Tight_noQGL_binaryCSV_v1.0.2 -f $SIMPLETOPTAGGERFILE -d $TAGGERCFGDIR
+# -o for overwrite softlinks if they exist
```

Comment:
I don't think we need to overwrite if they exist. We use this to setup environment every time we log in. Once the link is created it, that is fine.